From f76acdae162e77437582ea93856bb4f81109a359 Mon Sep 17 00:00:00 2001
From: Markus Vieth <39675748+mvieth@users.noreply.github.com>
Date: Sat, 17 Jan 2026 15:50:25 +0100
Subject: [PATCH] Replace deprecated VTK function

On the VTK master branch, the function is already removed.
Related to https://github.com/PointCloudLibrary/pcl/pull/6314/commits/7e4f6e844bfd1a54a61337324ae23cf49113e800
---
 visualization/src/histogram_visualizer.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/visualization/src/histogram_visualizer.cpp b/visualization/src/histogram_visualizer.cpp
index 3be17dd8c44..c537a14e472 100644
--- a/visualization/src/histogram_visualizer.cpp
+++ b/visualization/src/histogram_visualizer.cpp
@@ -170,7 +170,7 @@ void
 pcl::visualization::PCLHistogramVisualizer::reCreateActor (
     const vtkSmartPointer<vtkDoubleArray> &xy_array, RenWinInteract* renwinupd, const int hsize)
 {
-  renwinupd->ren_->RemoveActor2D (renwinupd->xy_plot_);
+  renwinupd->ren_->RemoveViewProp (renwinupd->xy_plot_);
   renwinupd->xy_plot_->RemoveAllDataSetInputConnections ();
   
   double min_max[2];
